ucp/_libs/ucx_api.pyx

```diff
@@ -111,7 +111,7 @@ cdef dict ucx_config_to_dict(ucp_config_t *config):
         if fflush(text_fd) != 0:
             clearerr(text_fd)
             raise IOError("fflush() failed on memory stream")
-        py_text = text.decode()
+        py_text = text.decode("ISO-8859-1")
```

Comment:
Interesting why this ISO version? Typically this is just something like `"ascii"`. Does that work here?

Reply:
It doesn't:

```python
UnicodeDecodeError: 'ascii' codec can't decode byte 0xff in position 89: ordinal not in range(128)
```

It seems that `"ascii"` just comprises 7-bit, while `"iso-8859-1"` is 8-bit.

Reply:
Is it possible this is a typo in the config? Decoding this byte with this encoding gives...

```python
In [1]: b"\xff".decode("iso-8859-1")
Out[1]: 'ÿ'
```

Seems a little odd (admittedly I don't know the context)

Reply:
Yeah, I don't think they will necessarily be parsed correctly. Those issues come from UCX output that's usually passed to the command-line, so they might very well be just special terminal characters, but we need the decode to not actually raise an exception. This is perhaps not the optimal solution -- ideally we would convert/drop those -- but I don't think it's worth spending much time on that now.

Reply:
If they are terminal characters and we just want to ignore the exception, would suggest we do that instead of trying to decode them to something that doesn't make sense. Would suggest a change like the following

```suggestion
        py_text = text.decode(errors="ignore")
```

Reply:
For context, this is how that flag behaves

```python
In [1]: b"abc\xff".decode()
---------------------------------------------------------------------------
UnicodeDecodeError                        Traceback (most recent call last)
<ipython-input-1-f49fbbb5f271> in <module>
----> 1 b"abc\xff".decode()

UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 3: invalid start byte

In [2]: b"abc\xff".decode(errors="ignore")
Out[2]: 'abc'
```

Reply:
Yeah, that's a good idea, the latest commit uses your suggestion.